lzero/entry/train_muzero.py

```diff
@@ -216,6 +239,11 @@ def train_muzero(
                 logging.info(f'eval offline finished!')
             break
 
+    # Remove hooks after training.
```

Comment:
implement this part in the `__del__` method of policy

Reply:
done